File: app/src/main/java/com/vereshchagin/nikolay/stankinschedule/ui/settings/editor/subsection/SettingsNotificationFragment.java
package com.vereshchagin.nikolay.stankinschedule.ui.settings.editor.subsection;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import com.vereshchagin.nikolay.stankinschedule.R;

/**
 * Категория настроек уведомлений приложения.
 */
public class SettingsNotificationFragment extends PreferenceFragmentCompat {

    public SettingsNotificationFragment() {
        super();
    }

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.preferences_notification, rootKey);
    }
}

File: legacy/src/main/java/com/vereshchagin/nikolay/stankinschedule/ui/settings/view/notification/SettingsNotificationFragment.kt
package com.vereshchagin.nikolay.stankinschedule.ui.settings.view.notification

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.vereshchagin.nikolay.stankinschedule.R

/**
 * Категория настроек уведомлений приложения.
 */
class SettingsNotificationFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle, rootKey: String) {
        setPreferencesFromResource(R.xml.preferences_notification, rootKey)
    }
}